### PR TITLE
Milestone 2: Code refactor using class

### PR DESCRIPTION
**Pull request by Lizdev-05**
In this milestone, I achieved the following...

- Refactor the code using classes
- Give the code overall styling
- Add background image

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -36,45 +9,91 @@ function errorMessage(message) {
...
-    errorMessage('Kindly fill all inputs');
-  } else {
-    books.push(bookList);
     localStorage.setItem('books', JSON.stringify(books));
-    displayBooks(bookList.id, bookList.title, bookList.author);
   }
 }
 
-// Global variable to check local storage
+// const book = new Book();
```
Comment (BenMukebo) [inline app.js]: - [ ] Kindly remove all commented line code, It is always a good practice to keep your code clean from such comments. 😉 💯

File: app.js [hunk 2]
```diff
@@ -36,45 +9,91 @@ function errorMessage(message) {
...
+  const items = document.createElement('li');
+  items.innerHTML += `
+  <div class="title-div"><h2><q>${title}</q></h2>
+  <h2> by </h2>
+  <h2> ${author}</h2></div>
+     `;
+  const removeBtn = document.createElement('button');
+  removeBtn.textContent = 'Remove';
+  removeBtn.className = 'removeBtn';
+  // removeBtn.id = id;
```
Comment (BenMukebo) [inline app.js]: - [ ] Kindly remove all commented line code, It is always a good practice to keep your code clean from such comments. 😉 💯

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @Lizdev-05, @Abdullah2213565,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- No linter errors. ✔️ 
- GitHub flow was correctly used.  ✔️   
- It has class methods to add and remove books✔️ 
- It keeps all functionality of the application ✔️ 
- It uses CSS styles to make the application match [this wireframe](https://github.com/microverseinc/curriculum-javascript/blob/bb245ca8a9d3bb04ebefbbd45e800764e1fc5f74/books/images/awesome_books_core_elements.png) ✔️  
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


- [ ] This is a group project. Please let's include all the authors in the README file. The Getting Started section and the License section of [this template](https://github.com/microverseinc/readme-template) are missing in your README file. Please let's make sure that all the main headers in the template are present in your README file. This makes the README file professional and descriptive. 

- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.js (hunk 1 above)
Comment (Lizdev-05) [reply app.js]: Thank you @BenMukebo for your feedback. I will work on the author that has  not been added to this particular branch, but for the "getting started" section, I had already modified it to suit my project, I don't think it should be listed directly as it was in the template( moreover, the template is a guide and not to copy and paste)
 secondly, for the MIT license, It is included already, I don't know if it is an oversight from your end.

I will remove the redundant code, work on the second author, and revert. Thanks

**Review by prolajumokeoni — APPROVED** [review]
Hi @Lizdev-05, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀 
- Good design 💯 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


- Kindly ensure that as partners you always have balanced commits in a pair programming project 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.js (hunk 1 above)
Comment (BenMukebo) [reply app.js]: Hey @Lizdev-05, Sorry for the MIT license,  it was an oversight from, my end. 😄

Comment (Lizdev-05) [reply app.js]: Okay, Thank you so much for the review.

File: app.js (hunk 2 above)
Comment (Lizdev-05) [reply app.js]: I will check it out. Thank you.

**Comment by Lizdev-05** [comment]
Thanks for reviewing and approving my project, I appreciate it.

**Pull request merged**